java/src/main/java/com/saucelabs/saucebindings/visual/SauceVisualOptions.java

```diff
@@ -0,0 +1,43 @@
+package com.saucelabs.saucebindings.visual;
+
+import com.saucelabs.saucebindings.SauceEnvironmentVariablesNotSetException;
+import org.openqa.selenium.MutableCapabilities;
+
+public class SauceVisualOptions {
+    private final MutableCapabilities visualCapabilities;
```

Comment:
Codacy found an issue: [Perhaps 'visualCapabilities' could be replaced by a local variable.](https://app.codacy.com/manual/SauceLabs/sauce_bindings/pullRequest?prid=6239731)